.github/workflows/ci.yml

```
@@ -1,6 +1,12 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Ah, excellent! Yes, it would be great to get CI for release branches as well. I assume this is how to do it?

```suggestion
      - master
      - release/*
```